launcher/FileSystem.cpp

```diff
@@ -770,9 +770,37 @@ QString getDesktopDir()
 bool createShortcut(QString destination, QString target, QStringList args, QString name, QString icon)
 {
 #if defined(Q_OS_MACOS)
-    destination += ".command";
+    // Create the Application
+    QDir applicationDirectory = QStandardPaths::writableLocation(QStandardPaths::ApplicationsLocation) + "/Prism Launchers/";
```

Comment:
haven't tested this yet, but is this $HOME/Applications/ or /Applications/ ?

Reply:
When I have tested it, it returns $HOME/Applications